test/e2e/config/azure-dev.yaml

```
@@ -32,8 +32,8 @@ providers:
           new: --metrics-addr=:8080
       files:
         - sourcePath: "../data/shared/v1alpha4/metadata.yaml"
-    - name: v1.1.4
-      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.1.4/core-components.yaml
+    - name: v1.2.0-beta.1
```

```suggestion
    - name: v1.2.0
```
I think you have to use v1.2.0 instead. clusterctl uses the latest non-prerelease tag (same for CABPK/KCP).
And it then later finds out that that tag (v0.4.7) doesn't have the right contract

---

No idea about the image pull issue though, but this solves the clusterctl issue later on

---

Looks like the pull issue was there before: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2425/pull-cluster-api-provider-azure-e2e/1541449105413246976
Maybe something that should be investigated independently (or is it maybe expected?)

---

thx!, done